src/OrchardCore/OrchardCore.DisplayManagement.Abstractions/DisplayType.cs

```diff
@@ -0,0 +1,12 @@
+namespace OrchardCore.DisplayManagement;
+
+public sealed class DisplayType
```

Comment:
Why not put this class on OrchardCoreConstants?

Reply:
I thought about that at the beginning, but then I realized this is a display management related thing

Reply:
OrchardCoreConstants already contains various of things like Roles, Shell Configuration. I think it is fair to put it there also just so we don't have so many files with constants.

Reply:
Ok